relayer/client.go

```diff
@@ -186,12 +186,8 @@ func CreateClient(ctx context.Context, src, dst *Chain, srcUpdateHeader, dstUpda
 	// the dst chains implementation of CreateClient, to ensure the proper client/header
```

Comment:
That logic is the same, the difference is that the assembly method is called on `src` instead of `dst` now on line 190. All message assembly functions are now called on the chain they intend to be sent to. This should help the composable and others with their integration since they can implement `MsgCreateClient` as the method to construct a message to be sent to the composable chain.

Reply:
Thanks for the clarification!